src/Core/Commander/Providers/PanoramicProvider.js

```diff
@@ -72,24 +70,18 @@
      * @param {type} distance
      * @returns {Promise}
      */
-    PanoramicProvider.prototype.getMetaDataFromPos = function(longitude, latitude, distance){
-
-        var that = this;
-        if(!_panoramicsMetaData){
+    PanoramicProvider.prototype.getMetaDataFromPos = function(longitude, latitude/*, distance*/){
```

Comment:
Wasn't sure what to do here: entirely remove the unused `distance` parameter or just comment it out?

FWIW, it was passed to `getClosestPanoInMemory` which didn't _expect_ it; and I inlined `getClosestPanoInMemory` here in the promise's `then`.